lib/command/diff.php

```diff
@@ -0,0 +1,158 @@
+<?php
+
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class rex_ydeploy_command_diff extends rex_ydeploy_command_abstract
+{
+    protected function configure()
+    {
+        $this
+            ->setName('ydeploy:diff')
+            ->setDescription('Updates schema file and creates diff file if necessary')
+            ->addOption('empty', null, InputOption::VALUE_NONE, 'Create (empty) diff file even if nothing has changed')
+            ->addOption('unmarked', null, InputOption::VALUE_NONE, 'Do not mark the diff file as already executed')
```

Comment:
Hmm, passt für mich hier nicht. Wir sind hier ja im `diff`-Kommando, nicht in `migrate`.
Normalerweise, wenn ich ein `diff` erstelle, braucht es bei mir selbst ja nicht mehr migriert werden, da wir ja ausgehen von der gerade bei mir aktuellen DB-Struktur.
Daher markiere ich normalerweise die erstellten Diffs beim Ersteller direkt als bereits ausgeführt.
Es kann aber Fälle geben, wo man das nicht will, insbesondere wenn man ein leeres Diff erstellt.

`dry-run` würde für mich bedeuten, es wird nur geprüft, ob ein Diff erstellt werden würde, es wird aber nicht wirklich erstellt. Könnte man eventuell auch irgendwann mal ergänzen die Option.
Meine Option oben meinte aber: Diff wird erstellt, aber es wird nicht als bereits ausgeführt markiert.